hazelcast-client-new/src/main/java/com/hazelcast/client/config/XmlClientConfigBuilder.java

```diff
@@ -167,15 +169,6 @@ public ClientConfig build(ClassLoader classLoader) {
         }
     }
 
-    ClientConfig build(ClientConfig clientConfig) {
```

Comment:
why did you remove this method? actually, it is `package-private`, so not public. also we have similar method `build(Config config)` in server-side.